test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -39,6 +39,7 @@ import (
 )
 
 var ignoreCQConditions = cmpopts.IgnoreFields(kueue.ClusterQueueStatus{}, "Conditions")
+var ignoreLastChangeTime = cmpopts.IgnoreFields(kueue.ClusterQueuePendingWorkloadsStatus{}, "LastChangeTime")
```

Comment:
I think we should add some integrations tests here. The important scenarios: 
- update of the pending workloads when a new workload is scheduler, 
- update of the pending workloads then a workload is admitted
- cut off the head of pending workloads when the number of pending workloads exceeds MaxCount. I think for the ease of testing we can setup the feature with MaxCount=3 (for example).